README.md

```diff
@@ -1 +1,202 @@
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Web-development-course-Project] <a name="about-project"></a>
+
+
+
+**[Web-development-course-Project]** is a project of making a portfolio website to showcase 
+my skills to  employers that i can make a responsive website using html, css, Javascript and Bootstrap.
+
+
+- [Video Link about Project] <a href="https://loom.com/share/b9814f9ce7c5488e91b1aa0b2a93a3fe"><button type="button" color:red background: white>Click Here</button></a>
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+This project has been made using html, css, javascript and bootstrap according to project requirements.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com/">html.com</a></li>
+    <li><a href="https://www.w3.org/TR/html401/present/styles.html">w3.org</a></li>
+  </ul>
+</details>
+
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+
+- **[Home_Page_is_Designed]**
+- **[About_Section_is_Designed]**
+- **[Feature_Section_is_Designed_using_javascript]**
+- **[Responsive_version_is_Made]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<!-- Note: Right now Application hasn't been Deployed yet. In future if this will be deployed,
+Deployed Link will update here -->
+- [Live Demo Link]  <a href="#"><button type="button" color:red background: white>Visit Site</button></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+A new developer could make use of this project by cloning the repository and could make adjustments
+in the code accordingly.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+1- Install Visual Studio Code<br>
+2- Install Live server Extension in Visual studio code
+ 
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:/zunairkhan811/Web-development-course-Project.git
+```
+
+
+### Install
+
+Install this project with:
+
+
+--> Visual Studio Code
+
+
+### Usage
+
+To run the project, execute the following command:
+
+
+--> Click on live Link or Click on live server in Vs Studio.
+
+
+### Run tests
+
+To run tests, run the following command:
+
+<!--
+In order to run for linter errors, you will have to create a pull request
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+
+--> By purchasing domain and hosting
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Author**
```

Comment:
- [ ] Please since you are the only contributor to this project you can singularize the `Authors` to `Author` and replace `Author` with your name. :smile: